src/main/java/com/gb/cornucopia/cookery/Cookery.java

```diff
@@ -2,8 +2,8 @@
 
 import com.gb.cornucopia.CornuCopia;
 import com.gb.cornucopia.InvModel;
-import com.gb.cornucopia.bees.Bees;
-import com.gb.cornucopia.cheese.Cheese;
+import com.gb.cornucopia.farming.bees.Bees;
```

Comment:
~tbh, these extra namespace layers should have been left until after updating to 1.12. I obiously can't force you to go back and change them, but I would like to reiterate: don't change unnecessary things until the update is complete.~

EDIT: I take it back: please go back and re-rename the modules so the old namespaces are preserved. github wasn't smart enough to know they were renames, so every change after this one has been split into two separate files.

Reply:
to be clear, I'm talking about the `.food` and `.farming` modules.